ajax/libs/mrnode/1.0.2/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "mrnode",
+  "version": "1.0.2",
+  "description": "A Javascript Library for Node.js Functions",
+  "main": "dist/js/main.js",

```

Comment:
@talonbragg We don't need `"main"`, `"scripts"`, `"bugs"`, `"devDependencies"`, `"dependencies"`and `"directories"`. Let's remove it.